internal/characters/lyney/cons.go

```diff
@@ -0,0 +1,161 @@
+package lyney
+
+import (
+	"github.com/genshinsim/gcsim/pkg/core/attacks"
+	"github.com/genshinsim/gcsim/pkg/core/attributes"
+	"github.com/genshinsim/gcsim/pkg/core/combat"
+	"github.com/genshinsim/gcsim/pkg/core/event"
+	"github.com/genshinsim/gcsim/pkg/core/glog"
+	"github.com/genshinsim/gcsim/pkg/core/player/character"
+	"github.com/genshinsim/gcsim/pkg/enemy"
+	"github.com/genshinsim/gcsim/pkg/modifier"
+)
+
+const (
+	c1ICDKey = "lyney-c1-icd"
+	c1ICD    = 15 * 60
+)
+
+// Lyney can have 2 Grin-Malkin Hats present at once.
+// Additionally, Prop Arrows will summon 2 Grin-Malkin Hats and grant Lyney 1 extra stack of Prop Surplus.
+// This effect can occur once every 15s.
+func (c *char) c1() {
+	if c.Base.Cons < 1 {
+		return
+	}
+	c.maxHatCount = 2
+}
+
+func (c *char) c1StackIncrease() int {
```

Comment:
why is it called `c1StackIncrease`?

Reply:
See the C1 text. At C1, every 15s the prop stack increase via prop arrow becomes 2 instead 1. This function determines that.

Reply:
but there's not just c1 stack here.